tools/depends/target/gnutls/Makefile

```diff
@@ -1,9 +1,9 @@
 include ../../Makefile.include
-DEPS= ../../Makefile.include Makefile size-max.patch 02-darwin-getentropy.patch
+DEPS= ../../Makefile.include Makefile size-max.patch 02-darwin-getentropy.patch remove-weak_import-check-for-osx.patch
```

Comment:
Get entropy patch should be removed. I don’t think it’s required using this version is gnutls.

Reply:
Ref: https://github.com/xbmc/inputstream.ffmpegdirect/pull/5/files

Reply:
@phunkyfish the original PR for that is here https://github.com/xbmc/xbmc/pull/13479

Are you saying the original issue isn't present anymore or has it been fixed in gnutls?

Reply:
I no longer require it when building gnutls in the inutstream addon. They have added logic for apply to then section of code in the newer release. Would be worth checking does it still build without that patch.

Reply:
I'll drop it and see what happens, thanks!